### PR TITLE
Anniversary Meetup 12-Jan-2019 blog

### PR DESCRIPTION
**Pull request by Kaushl2208**

### CODE REVIEW

File: content/2019-01-12-Anniversary-meetup.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+After the first few talks, we were asked to have a Break and Networking Session, which was really fun.
+
+
+After which Arif Khan gave a very informative talk on edge computing, where he discussed the future with edge computing and the differences between cloud computing and edge computing.This experience in the industry and an informative slide helps us to understand the topic lot more.
+
+![Anniversary-meetup](/images/12-01-19-Anniversary-meetup1.jpeg)
+
+
+After a quick coffee break, Vipin Rathi took a talk on the network technologies involved in edge computing the future scope of 5g technology and how it will affect the aspects of edge computing after which he also discussed the aspects of the blockchain.
```
Comment (bhanuvrat) [inline content/2019-01-12-Anniversary-meetup.md:21]: is it possible to link slides of the talk?

File: content/2019-01-12-Anniversary-meetup.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The last talk was presented by Abhimanyu Bhaater in which he describes L2 Switches in brief which was elaborated precisely.
+
+Volunteers meet was followed by hiring and pitching session where as usual everyone who all are looking for internships or jobs pitched for themselves. As this list will be posted on the community mailing list and LinkedIn. In the volunteer meet those who were willing to volunteer for Pydelhi were told about the responsibilites and area where they can help the community.
```
Comment (bhanuvrat) [inline content/2019-01-12-Anniversary-meetup.md]: Has this been done? If so, kindly add a link to the mailing list archive linking to the mail.

File: content/2019-01-12-Anniversary-meetup.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+
+We ended our special anniversary meetup with cake cutting ceremony which was flagged off with 8th -grade students. Having such youngsters in these meet-ups is really inspiring.Having a delicious cake in hand and like-minded funny and happy people all around made this meetup real success.
```
Comment (bhanuvrat) [inline content/2019-01-12-Anniversary-meetup.md]:
```suggestion
We ended our special anniversary meetup with cake cutting ceremony which was flagged off with 8th -grade students. Having such youngsters in these meet-ups is really inspiring. Having a delicious cake in hand and like-minded funny and happy people all around made this meetup real success.
```

File: content/2019-01-12-Anniversary-meetup.md (hunk 1 above)
Comment (Kaushl2208) [reply content/2019-01-12-Anniversary-meetup.md:21]: I don't think they gave the link of slide.Let me ask, if they have one I will update 😊 @bhanuvrat

Comment (rajataaron) [reply content/2019-01-12-Anniversary-meetup.md:21]: Let me ask Akshay if he can help us with the talk slides?

Comment (realslimshanky) [reply content/2019-01-12-Anniversary-meetup.md:21]: Let's post without the slides, it can be updated later on if possible.

**Comment by Kaushl2208** [comment]
All changes updated. @rajataaron  @realslimshanky

**Review by realslimshanky — COMMENTED** [review]
Please remove unnecessary blank lines from the md file

**Comment by Kaushl2208** [comment]
@realslimshanky Thank you for reviewing that. All suggestions are updated.

File: content/2019-01-12-Anniversary-meetup.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+Title: PyDelhi Anniversary Meetup at Microsoft Gurugram
+Date: 2019-01-12
+Category: Meetup
+Slug: pydelhi-meetup-12-January-2019
+Author: Kaushlendra Pratap 
+Email: kaushlendrapratap.9837@gmail.com
+
+#PYDELHI BLOG
```
Comment (realslimshanky) [inline content/2019-01-12-Anniversary-meetup.md]: Please remove this line.

File: content/2019-01-12-Anniversary-meetup.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+##PYDELHI & BLOCKCHAIN Devs ANNIVERSARY MEETUP AT Microsoft Gurugram
```
Comment (realslimshanky) [inline content/2019-01-12-Anniversary-meetup.md]: Please remove this line.

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Because of the lines I mentioned in the review below the website is not rendering correct navigation on the homepage which currently looks like

image


It should be
image

**Comment by Kaushl2208** [comment]
@realslimshanky All requested changes are updated. Thank you for the suggestions.

**Comment by Kaushl2208** [comment]
@realslimshanky  All the changes are updated. Thank you once again for reviewing that.

**Review by satyaakam — APPROVED** [review]

**Pull request merged**